client/components/button/style.scss

```diff
@@ -122,7 +122,8 @@ button {
 		color: $white;
 	}
 	&[disabled],
-	&:disabled {
+	&:disabled,
+	&.disabled {
 		color: lighten( $gray, 30% );
 		background: $white;
 		border-color: lighten( $gray, 30% );
```

Comment:
What do you think about adding the following styling to the top of this file?
```scss
.button {
    transition: color linear 0.1s, background-color linear 0.1s, border-color linear 0.1s
}
```

Reply:
That's quite a big change that will impact all buttons across calypso. Better to open a new PR for that if you want to suggest it I think